ios/Nodes/REASetNode.m

```diff
@@ -10,8 +12,20 @@ @implementation REASetNode {
 - (instancetype)initWithID:(REANodeID)nodeID config:(NSDictionary<NSString *,id> *)config
 {
   if ((self = [super initWithID:nodeID config:config])) {
-    _whatNodeID = config[@"what"];
-    _valueNodeID = config[@"value"];
+    _whatNodeID = [RCTConvert NSNumber:config[@"what"]];
+    if (_whatNodeID == nil) {
```

Comment:
In objC common pattern is to use macros for these cases

Reply:
Check out this macro, it's the first one I've written 👨‍💻